_posts/2022-04-12-identity.markdown

```diff
@@ -0,0 +1,81 @@
+---
+layout: post
+title:  "How Flotta project proofs identity on the far edge"
+date:   2022-04-12 12:00:00 +0100
+categories: flotta
+author: Eloy Coto <eloy.coto@acalustra.com>
+tags:
+  - authentication
+  - flotta
+
+summary-1: How Flotta project proofs identity on the far edge
+---
+
+When speaking about the edge one of the most problematic thing is the security.
```

Comment:
Either "one of the most problematic things" or "the most problematic thing"

Reply:
This is my fault :(
I wrongly suggested to replace `things` with `thing`